RELEASE.md

```diff
@@ -0,0 +1,71 @@
+## Release Documentation
+
+The following will cover releasing the `asset-transfer-api` to both github and NPM.
+
+### Creating The Release Branch
+
+1. Ensure you are in your `main` branch then run:
+
+```bash
+$ git pull origin main
+$ git checkout -b <name>-vXX-XX-XX
+```
+
+2. In your new release branch update the version inside of the `package.json` to match the version we are releasing. This is super important as it will be the source of truth for updating to NPM. 
+
+3. Inside of the `CHANGELOG.md`, follow the previous format of other releases, and input each commit that is being added in this current release. 
+
+4. Run `yarn docs` to update the docs and ensure they are correct.
+
+5. Run:
+
+```bash
+$ git add .
+$ git commit -m 'chore(release): vXX.XX.XX'
+$ git push
+```
+
+Then create a PR with the same title as the commit message.
+
+6. When approved merge the PR into main.
+
+### Creating the NPM release
+
+REQUIREMENTS: 
+- You must be part of the integrations NPM team for substrate. 
+- You must have 2FA enabled.
+
+1. Ensure your current working branch is `main`, and run the following as a sanity check.
+
+```bash
+$ git pull origin main
+$ yarn build
+$ yarn test
+```
+
+2. Run:
+
+```bash
+$ yarn deploy
+```
+NOTE:
+
+- Ensure the logging provided by NPM says the current version that is going to get released is correct before you enter your OTP.
+- If you are on node version 18 or below, it will ask you for your OTP in the terminal. But if you are on node 20 or greater if will redirect you to the browser to input the OTP. 
```

Comment:
```suggestion
- If you are on node version 18 or below, it will ask you for your OTP in the terminal. But if you are on node 20 or greater it will redirect you to the browser to input the OTP. 
```